.travis.yml

```diff
@@ -13,9 +13,10 @@ jdk:
   - oraclejdk8
 script:
   - if [[ "${TRAVIS_PULL_REQUEST}" = "false" ]]; then
-      export END_TASK="uploadArchives sonarqube";
+      export END_TASK="uploadArchives";
     fi
-  - ./gradlew clean assemble test ${END_TASK} --refresh-dependencies -Dsonar.host.url=https://sonarqube.com -Dsonar.login=$SONAR_TOKEN
+  - ./gradlew clean jacocoTestReport coveralls
+  - ./gradlew clean assemble ${END_TASK}
```

Comment:
merge 18 and 19 into one: `./gradlew clean assemble jacocoTestReport coveralls ${END_TASK}`. Once you get that, feel free to merge.